frontend/ui-pablo/app/components/Organisms/auction/AuctionDetails/ContractDetails.tsx

```diff
@@ -69,14 +69,14 @@ export const ContractDetails: React.FC<ContractDetailsProps> = ({
             </Typography>
             <Box display="flex" alignItems="center" gap={1.5} mt={2}>
               <Typography variant="subtitle1">
-                {`${asset?.symbol} on polkascan`}
+                {`${baseAsset?.symbol} on polkascan`}
```

Comment:
```suggestion
                {`${baseAsset?.symbol ?? ""} on polkascan`}
```
Fallback of empty string